Automattic/wp-calypso
# Update designCarousel to include link-in-bio related changes.

Related to #76719

## Proposed Changes

* DesignCarousel and Pattern-picker packages have similar functionality, so it is best to try to consolidate them for easier maintainability. This PR updates the designCarousel package to work with link-in-bio flow. 

## Testing Instructions



* Checkout this branch
* Execute `yarn start`
* Navigate to http://calypso.localhost:3000/setup/ecommerce/designCarousel to ensure it is working as intended and looks correct.
* Navigate to http://calypso.localhost:3000/setup/link-in-bio/patterns to ensure it is working as intended and looks correct.
* Navigate to http://calypso.localhost:3000/setup/link-in-bio-tld/patterns to ensure it is working as intended and looks correct.

## Pre-merge Checklist
